### PR TITLE
Melodic migration

### PR DESCRIPTION
**Pull request by rkojcev**

### CODE REVIEW

**Comment by jrgnicho** [comment]
@rkojcev Thanks for the PR I'll build it in a docker container and get back

**Comment by rkojcev** [comment]
@jrgnicho were you able to check if the plugin works good in melodic? if you need any help from my side please let me know.

**Comment by adaruna3** [comment]
@rkojcev I just compiled for melodic and it worked first try. Thanks!

File: moveit_cartesian_plan_plugin/CMakeLists.txt [hunk 1]
```diff
@@ -34,22 +58,27 @@ set(moveit_cartesian_plan_plugin_SRCS
...
 )
 
-set(moveit_cartesian_plan_plugin_HDRS
-  include/moveit_cartesian_plan_plugin/add_way_point.h
-  include/moveit_cartesian_plan_plugin/point_tree_item.h
-  include/moveit_cartesian_plan_plugin/point_tree_model.h
-  include/moveit_cartesian_plan_plugin/widgets/path_planning_widget.h
-  include/moveit_cartesian_plan_plugin/generate_cartesian_path.h
+set(${PROJECT_NAME}_HDRS
+  include/moveit_cartesian_plan_plugin/add_way_point.hpp
```
Comment (jrgnicho) [inline moveit_cartesian_plan_plugin/CMakeLists.txt:62]: Why change to hpp on all these header files?

**Comment by jrgnicho** [comment]
Thanks to all who contributed this PR

**Pull request merged**